### DISCUSSION TITLE
What to do

### DISCUSSION THREAD

**Discussion by Scribble24** [Drew-Alleman/DataSurgeon#9] [Q&A]
What to do pls. Running on ubuntu then this error.. 
![IMG20230512005813](https://github.com/Drew-Alleman/DataSurgeon/assets/127153735/3803dfd7-6271-4e35-a7b9-15596acbf72e)

**Comment by Drew-Alleman** [comment]
Did you just recently install ubuntu? I think you are missing build essentials. Run the commands below and try again.  
```
sudo apt update
sudo apt install build-essential
sudo apt upgrade
```

**Comment by Drew-Alleman** [comment]
Hi,

Did my last comment fix your problem? Please confirm/deny so I can keep this going or mark it as complete.

Best,
Drew

**Reply by Scribble24** [reply]
No it didn't. Just thought I may be causing you extra trouble that's why I say nothing.

**Comment by Drew-Alleman** [comment]
Hi,

This extra trouble is my hobby so feel free to keep writing back to me. Try these commands out.

```
sudo apt update --fix-missing
sudo apt install -y gcc cmake 
```

Best,
Drew
